AnkiDroid/build.gradle

```
@@ -387,7 +387,10 @@ dependencies {
     implementation libs.kotlinx.serialization.json
     implementation libs.seismic
 
-    debugImplementation libs.androidx.fragment.testing.manifest
+    // This uses implementation instead of debugImplementation to ensure
+    // fragment testing infrastructure is available in all build variants,
+    // not just debug builds, since androidTest on CI runs against both debug and release configurations.
+    implementation libs.androidx.fragment.testing.manifest
```

Sorry this sat so long! This seems to work for me in local testing. What do you think?

```suggestion
    // Fragment testing infrastructure has to be available during testing,
    // and we test release builds sometimes - include in main APK if so
    if (rootProject.testReleaseBuild) {
        implementation libs.androidx.fragment.testing.manifest
    } else {
        debugImplementation libs.androidx.fragment.testing.manifest
    }
```